bucket/crunchy-cli.json

```
@@ -1,25 +1,22 @@
 {
-    "version": "2.3.6",
-    "description": "A Crunchyroll cli client and downloader.",
+    "version": "3.0.0",
+    "description": "Command-line downloader for Crunchyroll",
     "homepage": "https://github.com/crunchy-labs/crunchy-cli",
-    "license": "GPL-3.0-or-later",
-    "suggest": {
-        "ffmpeg": "ffmpeg",
-        "vcredist": "extras/vcredist2010"
-    },
-    "notes": "You need a premium account for full access.",
+    "license": "MIT",
+    "depends": "ffmpeg",
```

```suggestion
    "depends": "ffmpeg",
```

`suggest` instead of `depends`.